src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/ServiceProviderContainerTests.cs

```diff
@@ -281,7 +281,7 @@ public void GetService_DisposeOnSameThread_Throws()
             });
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
This is interesting. Why wasn't this needed before?

Reply:
Because the entire base class was disabled on mono.

https://github.com/dotnet/runtime/blob/e348011cc618d1075c6f8ed2ffe378542f28e217/src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/ServiceProviderContainerTests.cs#L19
https://github.com/dotnet/runtime/blob/e348011cc618d1075c6f8ed2ffe378542f28e217/src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Specification.Tests/DependencyInjectionSpecificationTests.cs#L13-L14

I removed all those attributes on the base types in order to make the suppressions more granular.

Reply:
Makes sense. Thanks!